modules/node-shared/src/main/scala/io/constellationnetwork/node/shared/infrastructure/snapshot/CurrencySnapshotCreator.scala

```diff
@@ -271,7 +271,7 @@ object CurrencySnapshotCreator {
             currencySnapshotAcceptanceResult.sharedArtifacts.some,
             currencySnapshotAcceptanceResult.allowSpendBlock.accepted.toSortedSet.some,
             currencySnapshotAcceptanceResult.tokenLockBlock.accepted.toSortedSet.some,
-            currencySnapshotAcceptanceResult.globalSyncView.some
+            currencySnapshotAcceptanceResult.globalSyncView
```

Comment:
can this be required here? I thought schema changes were required to be optional for historic compatibility

Reply:
yes, exactly @scasplte2 :+1: